src/api.js

```diff
@@ -828,6 +854,7 @@ class VLWholeSlideMicroscopyImageViewer {
     const geometry = _scoord3d2Geometry(item.scoord3d, this._pyramid);
     const feature = new Feature(geometry);
     feature.setProperties(item.properties, true);
+    feature.setId(item.uid);
```

Comment:
Maybe we should generate the `id` here if `item.uid` is undefined?

Reply:
Addressed with a88b5879b0d3627c5430ca0bd348d32577b628fe